src/libs/actions/Report.js

```diff
@@ -364,6 +355,19 @@ function updateReportWithNewAction(reportID, reportAction) {
         updatedReportObject.lastActorEmail = reportAction.actorEmail;
     }
 
+    // When handling an action from the current users we can assume that their
+    // last read actionID has been updated in the server but not necessarily reflected
+    // locally so we must first update it and then calculate the unread (which should be 0)
+    if (isFromCurrentUser) {
+        setLocalLastRead(reportID, newMaxSequenceNumber);
+    }
+
+    // We update the lastReadSequenceNumber to reflect new.
+    lastReadSequenceNumber = lastReadSequenceNumbers[reportID] || 0;
```

Comment:
There's no need to reassign this unless it's from the current user so we can push it into the `isFromCurrentUser` block

Reply:
Done